tools/index.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+s=$(which mighty-batch)
+if [ "$s" = "" ];
+then
+    npm install -g mighty-batch
+    cd tools
+    npm install
+    cd ..    
+fi
+tar -zxf outdoors_posts.tar.gz
+mighty-batch --threads 1 --workers 1 --host mighty1 --json outdoors_posts.json --property Paragraphs
```

Comment:
I didn't found how to specify workers on different hosts, so I ended up enabling only a single node for indexing. 

I hope you know better how to fix that